src/panels/lovelace/cards/hui-entity-filter-card.js

```diff
@@ -72,35 +59,33 @@ class HuiEntitiesCard extends PolymerElement {
     if (this.lastChild) {
       this.removeChild(this.lastChild);
     }
-    let error = null;
-    let cardConfig;
-    let tag = config.card ? computeCardElement(config.card) : 'hui-entities-card';
+    let error;
+    let element;
 
-    if (tag === null) {
-      error = `Unknown card type encountered: "${config.card}".`;
-    } else if (!customElements.get(tag)) {
-      error = `Custom element doesn't exist: "${tag}".`;
-      if (!(tag in this._whenDefined)) {
-        this._whenDefined[tag] = customElements.whenDefined(tag)
-          .then(() => this._configChanged(this.config));
-      }
-    } else if (!config.filter || !Array.isArray(config.filter)) {
-      error = 'No or incorrect filter.';
+    if (!config.filter || !Array.isArray(config.filter)) {
+      error = 'Incorrect filter config.';
+    } else if (!config.card) {
+      config.card = { type: 'entities' };
+    } else if (!config.card.type) {
+      config.card.type = 'entities';
     }
+
+
     if (error) {
-      tag = 'hui-error-card';
-      cardConfig = { error };
+      element = document.createElement('hui-error-card');
+      element.error = error;
+      element.config = config;
     } else {
-      cardConfig = this._computeCardConfig(config);
+      element = createCardElement(config.card);
+      element.config = this._computeCardConfig(config);
+      element.hass = this.hass;
+      this._cardReady = true;
```

Comment:
If you set it to true here, you need to set it to false when there is an error. But I don't think that it adds much.